src/common/keyboard_layout.cpp

```diff
@@ -305,25 +305,25 @@ std::vector<DWORD> LayoutMap::LayoutMapImpl::GetKeyCodeList(const bool isShortcu
     return keyCodes;
 }
 
-std::vector<std::wstring> LayoutMap::LayoutMapImpl::GetKeyNameList(const bool isShortcut)
+std::vector<std::pair<DWORD, std::wstring>> LayoutMap::LayoutMapImpl::GetKeyNameList(const bool isShortcut)
 {
-    std::vector<std::wstring> keyNames;
+    std::vector<std::pair<DWORD, std::wstring>> keyNames;
     std::vector<DWORD> keyCodes = GetKeyCodeList(isShortcut);
     std::lock_guard<std::mutex> lock(keyboardLayoutMap_mutex);
     // If it is a key list for the shortcut control then we add a "None" key at the start
     if (isShortcut)
     {
-        keyNames.push_back(L"None");
+        keyNames.push_back({ 0, L"None" });
```

Comment:
Is the "None" string shown in the UI?
If yes, we should move it to the resx file, if not, let's replaced with a const and add a comment to explicitly say it's not a localizable string.

Reply:
Yes, it is shown in the keys dropdown as well as other keys(Enter, Shift, Disable, F1, Play...). We do not localize keys so far. 
https://github.com/microsoft/PowerToys/blob/d1372af581d74298447d14ad7116b6f7f4a1a04b/src/common/keyboard_layout.cpp#L104-L218
So it is part of a bigger work item.

Reply:
OK, let's not block this PR because of that, but please open an issue listing those keys since they will have to be localized.

Reply:
https://github.com/microsoft/PowerToys/issues/7304